questions.json

```diff
@@ -13,6 +14,22 @@
     "What is your least favorite friend's middle name?",
     "What is a question an oyster would ask a fish?",
     "What would your full name be if you married your first crush?",
-    "What caused you to realize that capitalism is a sham?"
+    "What caused you to realize that capitalism is a sham?",
+    "What is your favorite security question?",
+    "What are you wearing at time time of setting up the security question?",
```

Comment:
Typo here, "time time," this one might be kind of difficult to remember the answer to, also